tekton/resources/images/overlays/multi-arch/image-build-trigger-template.yaml

```diff
@@ -0,0 +1,70 @@
+- op: add
+  path: "/spec/resourcetemplates/0/spec/taskSpec/steps"
```

Comment:
NIT: I'm not a huge fan of the idea of injecting steps into a common task here, as it may make it more difficult to make changes to one of the tasks later on (like switching to catalog tasks, or building tasks to a bundle), but that's also something we can also change later when the time comes.

Reply:
I was thinking if we can have 2 separate sets for multi-arch and single-arch cases. But decided that having just separate code, where many parts are the same would lead even to more problems, when something needs to be updated for both cases. 
However if based on your experience it's better to do, I'd be happy to follow this path